pkg/test/framework/scope.go

```diff
@@ -104,6 +102,16 @@ func (s *scope) get(ref interface{}) error {
 		}
 	}
 
+	if s.parent != nil {
```

Comment:
Is this a bug fix? If so, mind splitting it out to its own PR?

Reply:
https://github.com/istio/istio/pull/25134